docs/UserGuide.md

```diff
@@ -122,7 +122,7 @@ Clears all students, tasks and lessons from JARVIS.
 
 Format: `clear`
 
-**Warning! This action is irreversible! Please think twice before you use this**
+#### **Warning! This action is irreversible! Please think twice before you use this!**
```

Comment:
This change was intentional. See [this comment](https://github.com/AY2223S1-CS2103T-T11-3/tp/pull/112#issuecomment-1293381591) by @fongyj

Reply:
oh sorry i just wanted to add the exclamation mark at the back, I didn't realise I added the header at the front as well